### ISSUE TITLE
A11y_Model Builder_Add Data_Data Preview_Keyboard : Graph controls are not accessed with keyboard.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2184]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev1; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Install Visual studio and open Visual studio
2. Navigate to C# project name and right click on it
3. Navigate to "Add" button and press enter
4. Navigate to "Machine learning model" and press enter
5. "Add new item" popup will open
6. Navigate to "Add" button and press enter
7. "Select scenario" popup will open and select "Forecasting" scenario
8. "Select training environment" page will open then select "Next step"
9. "Add Data" page will appear browse and upload the sample data file.
10.  tab till Data Preview section.
11. Tab till Graph radio button and press enter.
12. Check whether the graph controls are not accessed with keyboard or not.

### Actual Result:
Graph controls are not accessed with keyboard.

### Expected Result:
All the Graph controls should be accessed with keyboard.

![Screen Recording](https://user-images.githubusercontent.com/91586101/176697892-fd5bda19-4722-4f01-a28e-6195420cb127.gif)

**Comment by beccamc** [comment]
@LittleLittleCloud What are your thoughts on this? Can we get the keyboard focus into the graph? Or do we need to remove/disable those controls?

**Comment by LittleLittleCloud** [comment]
The graph is not a WPF control. It's using daytona to present js code. We probably need further investigation to see if it's available to get into graph using keyboard only.

For now I would sugguest just disable the access to graph as we already have table control. Any suggestions? @JakeRadMSFT

**Comment by beccamc** [comment]
Disable access to the graph? Or remove the controls in the upper right that allow the user to manipulate the graph?

**Comment by beccamc** [comment]
We don't actually want users to be able to manipulate this graph. We have removed the buttons (they were built in via the tool we're using to generate the graph). This is resolved as there are no longer buttons to interact with.

**Comment by manaswita-chougule** [comment]
Hi @beccamc could you please provide us the latest model builder environment, as we are still able to see the button controls of graph.
Thanks!

**Comment by beccamc** [comment]
@manaswita-chougule  Please verify this on build 16.13.10.2241902 available from the Marketplace https://marketplace.visualstudio.com/items?itemName=MLNET.ModelBuilder2022

We actually don't want users to be able to manipulate the graph. We have removed those utility buttons.

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment:
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.4.0 Preview 1.0

The Graph controls are removed, so it's non-interactive control. Hence there is no need for keyboard accessibility.
Closing the bug. Please find below attachment for reference.
![2184_keyboard_graph_fixed](https://user-images.githubusercontent.com/91586101/187126828-b9cb4baf-e40b-4195-9b14-26c20f1a6a8b.gif)
